src/CMakeLists.txt

```diff
@@ -1 +1 @@
-/opt/ros/kinetic/share/catkin/cmake/toplevel.cmake
\ No newline at end of file
+/usr/share/catkin/cmake/toplevel.cmake
```

Comment:
im assuming this is the contents of the symlink changing, but for me that new cmake file doesnt exist, and every time i run catkin_make this file is changed back to the old version